libp2p/connmanager.nim

```diff
@@ -62,24 +66,27 @@ type
     handle: Future[void]
 
   ConnManager* = ref object of RootObj
-    maxConns: int
-    # NOTE: don't change to PeerInfo here
-    # the reference semantics on the PeerInfo
-    # object itself make it susceptible to
-    # copies and mangling by unrelated code.
+    maxPeerConns: int
```

Comment:
this should be `maxConnsPerPeer` else it reads like a global limit on peers connected (regardless of direction)